airflow/www/views.py

```diff
@@ -2288,6 +2292,39 @@ class UserModelView(wwwutils.SuperUserMixin, AirflowModelView):
     column_default_sort = 'username'
 
 
+class VersionView(wwwutils.SuperUserMixin, LoggingMixin, BaseView):
+    @expose('/')
+    def version(self):
+        # Look at the version from setup.py
+        airflow_version = None
+        try:
+            airflow_version = pkg_resources.require("airflow")[0].version
+        except Exception as e:
+            self.logger.error(e)
+            pass
+
+        # Get the Git repo and git hash
```

Comment:
I don't think this will work for the vast majority of the people, since they just `pip install` Airflow, right? That's what we do. There is no git repo. Using AIRFLOW_HOME doesn't make sense to me either. We have AIRFLOW_HOME set to a location, again, with no `git` repo. Even if it *did* have a git repo, it wouldn't be the git repo that had the Airflow source in it, so the checksum would be no-sensical. 

Can I propose an alternative solution? Baking the git checksum into the Python package when it's installed (similar to __version__). It seems like this can be automated during the Python package build.